src/elysium/rules.cpp

```diff
@@ -131,6 +132,7 @@ CMainConsensusParams::CMainConsensusParams()
 
     // Sigma releated
     SIGMA_FEATURE_BLOCK = 212000; // 4 Nov 2019
+    SIGMA_SPENDV1_FEATURE_BLOCK = 258177; // 13 April 2020
```

Comment:
We need to update this due to a block number is already passed this value.

Reply:
You're right. Per the meeting, I would put it to 4 May 2020.